corehq/ex-submodules/pillowtop/reindexer/reindexer.py

```diff
@@ -26,15 +25,15 @@ def __init__(self, pillow, change_provider, elasticsearch, index_info):
     def reindex(self, start_from=None):
         if not start_from:
             # when not resuming force delete and create the index
-            self._delete_and_prepare_index_for_reindex()
+            self._prepare_index_for_reindex()
 
         super(ElasticPillowReindexer, self).reindex(start_from)
 
         self._prepare_index_for_usage()
 
-    def _delete_and_prepare_index_for_reindex(self):
-        ensure_production_index_deleted(self.index_info.index)
-        self.es.indices.create(index=self.index_info.index, body=self.index_info.meta)
+    def _prepare_index_for_reindex(self):
+        if not self.es.indices.exists(self.index_info.index):
+            self.es.indices.create(index=self.index_info.index, body=self.index_info.meta)
```

Comment:
I had thought of a user confirmation but I see this class is isolated from the management command. Maybe not worth the effort but you could use a `force` param and raise an exception if it's `False`.